src/pages/sighting/SightingCore.jsx

```diff
@@ -0,0 +1,279 @@
+import Link from '../../components/Link';
+import Text from '../../components/Text';
+import LoadingScreen from '../../components/LoadingScreen';
+import SadScreen from '../../components/SadScreen';
+import Button from '../../components/Button';
+import Alert from '../../components/Alert';
+import MoreMenu from '../../components/MoreMenu';
+import ConfirmDelete from '../../components/ConfirmDelete';
+import EntityHeaderNew from '../../components/EntityHeaderNew';
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import useDeleteSighting from '../../models/sighting/useDeleteSighting';
+import useCommitAssetGroupSighting from '../../models/assetGroupSighting/useCommitAssetGroupSighting';
+import useSightingFieldSchemas from '../../models/sighting/useSightingFieldSchemas';
+import { formatDate } from '../../utils/formatters';
+// import AnnotationsGallery from './AnnotationsGallery';
+// import IndividualsGallery from './IndividualsGallery';
+import Annotations from './Annotations';
+import Photographs from './Photographs';
+import OverviewContent from './OverviewContent';
+import SightingHistoryDialog from './SightingHistoryDialog';
+import FeaturedPhoto from './featuredPhoto/FeaturedPhoto';
+import Encounters from './encounters/Encounters';
+
+export default function SightingCore({
+  data,
+  loading,
+  error,
+  statusCode,
+  pending,
+  id,
+}) {
+  const history = useHistory();
+  const intl = useIntl();
+  const queryClient = useQueryClient();
+
+  const fieldSchemas = useSightingFieldSchemas();
+
+  function refreshData() {
+    const queryKey = pending
+      ? getAGSQueryKey(id)
+      : getSightingQueryKey(id);
+    queryClient.invalidateQueries(queryKey);
+  }
+
+  const {
+    deleteSighting,
+    loading: deleteInProgress,
+    error: deleteSightingError,
+    setError: setDeleteSightingError,
+  } = useDeleteSighting();
+
+  const {
+    commitAgs,
+    isLoading: commitAgsLoading,
+  } = useCommitAssetGroupSighting();
+
+  /*
+    known issue: if data or fieldschemas change values
+    or properties this may not update
+    switch to data.version?
+  */
+  const metadata = useMemo(
+    () => {
+      if (!data || !fieldSchemas) return null;
+      return fieldSchemas.map(schema => ({
+        ...schema,
+        value: schema.getValue(schema, data),
+      }));
+    },
+    [data, fieldSchemas],
+  );
+
+  // fetch data for Id...
+  useDocumentTitle(`Sighting ${id}`, { translateMessage: false });
+
+  const [historyOpen, setHistoryOpen] = useState(false);
+  const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
+
+  const activeTab = window.location.hash || '#overview';
+
+  if (loading) return <LoadingScreen />;
+  if (statusCode === 404)
+    return (
+      <SadScreen
+        subtitleId="SIGHTING_NOT_FOUND"
+        descriptionId="SIGHTING_NOT_FOUND_DESCRIPTION"
+        variant="genericError"
+      />
+    );
+  if (error) return <SadScreen variant="genericError" />;
```

Comment:
This feels like the least specific conditional. Should this not be the last one encountered? In other words, why not move this below the if(!data) section below?

Reply:
good call - moved.